spec/README.md

```diff
@@ -138,6 +138,15 @@ Type: `String`
 
 The associated language written in the native language.
 
+###### metadata.countries
+
+Optional
+Type: `Array`
+
+An array of countries where the given language is officially spoken. This entry is automatically added by the build script, with information obtained directly from the [languageData](https://github.com/unicode-cldr/cldr-core/blob/master/supplemental/languageData.json) and [territoryInfo](https://github.com/unicode-cldr/cldr-core/blob/master/supplemental/territoryInfo.json) provided by the unicode-CLDR database.
+
+The standard Unicode language identifiers follow [IETF BCP 47](https://tools.ietf.org/html/bcp47) with some small differences; please see the [CLDR spec](http://cldr.unicode.org/index/cldr-spec/picking-the-right-language-code) for more details. For a quick reference, use this [IETF-language-tags](http://data.okfn.org/data/core/language-codes) searchable table.
```

Comment:
I personally don't understand this paragraph. First of all, the content of the link "IETF BCP 47" is way too long, I'm not spending hours digging into what you meant here or comparing the content of the link with "CLDR spec". I think user will neither. The link to "IETF-language-tags" is helpful, but since it contains many tables you'll definitely have to use a `hash`. If it doesn't show up correctly, this is still better than not targeting a specific table. Finally, I think you should mention that you're referring to "territory" entries within that table, aren't you?